stories/radioButtonGroup.md

```diff
@@ -1,5 +1,3 @@
-# RadioButtonGroup
-
```

Comment:
Though we have actual docs here, this heading is still redundant with the Docs page heading.